jsgallery/main.js

```diff
@@ -0,0 +1,71 @@
+const hoverTarget = document.querySelectorAll('.roma_box');
+
+//마우스 호버 효과
+for (var i = 0; i < hoverTarget.length; i++) {
+    hoverTarget[i].addEventListener('mouseenter', onHover);
+    hoverTarget[i].addEventListener('mouseleave', outHover);
+}
```

Comment:
선언형 프로그래밍을 위해!
```suggestion
hoverTarget.forEach((target) => {
    target.addEventListener('mouseenter', onHover);
    target.addEventListener('mouseleave', outHover);
})
```
그리고 하나 하나 지금 돌아가면서 EventListener를 붙여주고 잇는데 조금 더 심화지만 [이벤트 위임](https://ko.javascript.info/event-delegation)을 이용하면 이렇게 하지 않아도 된답니당 ㅎㅎ 참고 ~

Reply:
유용유용❤ 팟짱의 코리 완전 땡큐💞